src/Symfony/Component/MimePgp/PgpMime.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\MimePgp;
+
+use Symfony\Component\Mime\Exception\RuntimeException;
+
+/*
+ * @author PuLLi <the@pulli.dev>
+ */
+abstract readonly class PgpMime
```

Comment:
this class should not be marked as `readonly` (or it should be `@internal`), as it prevents changing the whole class hierarchy forever (and also prevents using property hooks to write a BC layer for properties if needed, as readonly classes cannot use them).

Reply:
maybe this should be an internal trait instead.

Reply:
OK noted. I will change that 👍🏼.